sdk/go/pulumi/config/require.go

```diff
@@ -18,17 +18,21 @@ package config
 
 import (
 	"encoding/json"
+	"fmt"
 
 	"github.com/spf13/cast"
 
-	"github.com/pulumi/pulumi/sdk/v3/go/common/util/contract"
 	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
 )
 
+func failf(format string, a ...interface{}) {
+	panic(fmt.Sprintf(format, a...))
```

Comment:
Nit: I'd panic with an error here to make it a bit easier to recover if necessary.

```suggestion
	panic(fmt.Errorf(format, a...))
```